Makefile

```diff
@@ -5,6 +5,9 @@ IMAGE_TEST := reconcile-test
 IMAGE_NAME := quay.io/app-sre/qontract-reconcile
 IMAGE_TAG := $(shell git rev-parse --short=7 HEAD)
 
+POSTGRES_IMAGE_NAME := quay.io/app-sre/qontract-reconcile-postgres
+MDB_IMAGE_NAME := quay.io/app-sre/qontract-reconcile-mariadb
```

Comment:
Nit: `MARIADB_IMAGE_NAME ` instead of `MDB_IMAGE_NAME` for consistency and clarity

Reply:
Fixed